ios/Exponent/Versioned/Core/UniversalModules/Permissions/EXScopedPermissions.m

```diff
@@ -129,7 +129,7 @@ - (void)showPermissionRequestAlert:(NSString *)permissionType
                    withAllowAction:(UIAlertAction *)allow
                     withDenyAction:(UIAlertAction *)deny
 {
-  NSString *experienceName = self.experienceId; // TODO: we might want to use name from the manifest?
+  NSString *experienceName = self.experienceScopeKey; // TODO: we might want to use name from the manifest?
```

Comment:
I'll create a task to follow-up on this very soon. Will leave it scope key for now though since we don't really have anything better.
https://linear.app/expo/issue/ENG-1447/display-name-in-exscopedpermissions